<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacherControl</class>
 <widget class="QWidget" name="teacherControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>教师界面</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,10" columnstretch="1,4">
   <item row="1" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>课程号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_cid"/>
      </item>
      <item>
       <widget class="QPushButton" name="selectbutton">
        <property name="text">
         <string>确认查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>学号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_sid">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>成绩：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="input_grade">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>确认编辑</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>清空</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTableView" name="tableView"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
